### PR TITLE
Add BoxCam2D

### PR DESCRIPTION
**Pull request by sarooptech**

### CODE REVIEW

**Review by Calinou — CHANGES_REQUESTED** [review]
Please move the newly added line to follow alphabetical order within the list it was added to.

**Comment by sarooptech** [comment]
Any update on it ?

File: README.md [hunk 1]
```diff
@@ -163,6 +163,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 - [2D Destructible Objects](https://github.com/hiulit/Godot-3-2D-Destructible-Objects) - A script that takes a sprite, divides it into blocks and makes them explode💥 (Godot 3.x).
 - [AgonesSDK](https://github.com/AndreMicheletti/godot-agones-sdk) - Plugin to add [Agones](https://github.com/googleforgames/agones) SDK functionality to Godot (Godot 3.x).
 - [Anima](https://github.com/ceceppa/anima) - Run sequential and parallel animations with less code compared to Tween (Godot 3.x).
+- [BoxCam2D](https://github.com/samdaiya/BoxCam2D) - Simplest way to implement grid based 2D Camera movement (Godot 3.x).
```
Comment (Calinou) [inline README.md:166]:
```suggestion
- [BoxCam2D](https://github.com/samdaiya/BoxCam2D) - Simplest way to implement grid-based 2D camera movement (Godot 3.x).
```

File: README.md [hunk 2]
```diff
@@ -163,6 +163,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
...
 - [Aseprite Wizard](https://github.com/viniciusgerevini/godot-aseprite-wizard) - Plugin for importing animations from Aseprite as SpriteFrames (Godot 3.x).
```
Comment (Calinou) [inline README.md]:
```suggestion
- [Aseprite Wizard](https://github.com/viniciusgerevini/godot-aseprite-wizard) - Plugin for importing animations from Aseprite as SpriteFrames (Godot 3.x).
- [BoxCam2D](https://github.com/samdaiya/BoxCam2D) - Simplest way to implement grid-based 2D camera movement (Godot 3.x).
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**